portal-web/test/functional/com/liferay/portalweb/macros/KBArticle.macro

```diff
@@ -211,6 +211,54 @@ definition {
 		}
 	}
 
+	macro addCPWithExpirationDateProperty {
```

Comment:
Instead of creating a new macro please add a conditionals for expiration date to KBArticle.addCP